src/app/page/categoryInfo/deck/deck.component.ts

```diff
@@ -90,7 +90,7 @@ export class DeckComponent implements OnInit {
   changeDeckStatus(deckId: number) {
     for (const entry of this.decksWithStatus) {
       if (entry.deckId === deckId) {
-        entry.status = true;
+        entry.status = !entry.status;
```

Comment:
Optionally, I would add a break here to exit loop as soon as the required entry is found.

Reply:
You can commit to the same branch, the pull request should update automatically.